### ISSUE TITLE
Keyboard accessibility: Cannot check breakpoint checkboxes when selecting the line item

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#3379]
#3246

The debug panel focus goes first to the list, allowing selection of either "All Exceptions" or "Uncaught Exceptions", tabbing again will focus the All Exceptions checkbox, tabbing again will focus the Uncaught Exceptions checkbox. Focusing the list items seems to do nothing, I expect one of these to happen:

- The list items cannot be focuses/highlighted as toggling is done by focusing the checkboxes.
- You need to focus the list item and hit <kbd>enter</kbd> to toggle the checkbox.

**Comment by isidorn** [comment]
<kbd>enter</kbd> already navigates to that breakpoint. So I decided to use <kbd>space</kbd> instead.
I think it is common practice to use <kbd>space</kbd> for toggling state. Let me know how you feel about it.
